pkg/storage/couchdb/couchdbstore.go

```diff
@@ -312,6 +314,26 @@ func (c *CouchDBStore) Get(k string) ([]byte, error) {
 	return value, nil
 }
 
+// GetBulk fetches the values associated with the given keys. This method works in an all-or-nothing manner.
+// It returns an error if any of the keys don't exist. If even one key is missing, then no values are returned.
+// The end result is equivalent to calling Get(k,v) on each key-value pair individually in a loop,
+// but should be faster since this method minimizes REST calls as long as the values were stored as JSON instead of
```

Comment:
Note: In the EDV server, all of the values we'll be storing via `GetBulk` will be JSON, so the bulk attachment retrieval optimization that I mention here is irrelevant from the perspective of EDV server optimizations.